api/test_utils/flags/flags.go

```
@@ -0,0 +1,13 @@
+package flags
+
+import "flag"
+
+var (
+	Database   bool
+	Filesystem bool
+)
+
+func init() {
```

golangci-lint (1.64.8)

[error] don't use `init` function

(gochecknoinits)